docker-build-base.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh
 
 TAG=$( ./modelix-version.sh )
+MODELIX_TARGET_PLATFORM="${MODELIX_TARGET_PLATFORM:=linux/amd64}"
 
-docker build --no-cache -f Dockerfile-base -t modelix/modelix-base .
-docker tag modelix/modelix-base:latest "modelix/modelix-base:${TAG}"
\ No newline at end of file
+if [[ -z "${MODELIX_CI}" ]]; then
+  docker build --platform ${MODELIX_TARGET_PLATFORM} --no-cache -f Dockerfile-base \
```

Comment:
```suggestion
  docker build --platform "${MODELIX_TARGET_PLATFORM}" --no-cache -f Dockerfile-base \
```

Not really necessary here as we know the valid values. But it's generally a good idea to quote these variable expansions as otherwise empty values would result in no argument being passed. `shellcheck` is a good tool to find such issues in your editor.